examples/web-wagmi/src/profiles/UseFollowAndUnfollow.tsx

```diff
@@ -11,14 +11,12 @@ type FollowButtonProps = {
   profile: ProfileFragment;
 };
 
-const toastNotification = (error: Error) => toast.error(error.message);
-
 function FollowButton({ profile }: FollowButtonProps) {
   const { follow, isPending: isFollowing } = useFollow({ profile });
   const { unfollow, isPending: isUnfollowing, error } = useUnfollow({ profile });
 
   useEffect(() => {
-    if (error && 'message' in error) toastNotification(error);
```

Comment:
`'message' in error` caught my attention as it should not really be required when type definition is correct. 
Found the issue that `useUnfollow` was incorrectly referring to `IEquatableError` which is our internal error (strictly it's not an error as it's not extending `Error` due to type incompatibility).

Reply:
> it's not extending Error due to type incompatibility

what you mean by that?

Reply:
the `IEquatableError` has a custom implementation of `name` that's not a broad `string`
```
export interface IEquatableError<T extends string = string, P = Narrow<T>> {
  name: P;
}
```
the `Error` interface from `lib.d.ts` marks the error as just a `string`
```
interface Error {
    name: string;
    message: string;
    stack?: string;
}
```
given that we can't just do `interface IEquatableError<...> extends Error` as that would fail on the tsc level
```
TS2430: Interface 'IEquatableError<T, P>' incorrectly extends interface 'Error'.   Types of property 'name' are incompatible.     Type 'P' is not assignable to type 'string'
```

but anyway i don't remember any place in the code where we expose the `IEquatableError` to the sdk consumers, we always wrap it with a custom one that's easy to understand.

Reply:
Thank you for the explanation. Make sense.